backend/tests/unit/hume.test.js

```
@@ -0,0 +1,149 @@
+  default: {
+    from: jest.fn(() => ({
+      upsert: jest.fn().mockResolvedValue({ data: {}, error: null }),
+      update: jest.fn().mockResolvedValue({ data: {}, error: null })
+    }))
+  }
+}));
+
+describe('HumeService', () => {
+  it('throws if HUME_API_KEY is missing', async () => {
+    const original = process.env.HUME_API_KEY;
+    delete process.env.HUME_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)